### PR TITLE
Stringtable: Localise #709 strings

### PR DESCRIPTION
**Pull request by Jawshy**
##### Changes: 
- Added: Localisations for the new strings that were added in https://github.com/ArmaLife/Framework/pull/709. 
  - Translations via Google Translate. 
- Fixed: Tabs → spaces. 
  - As per https://github.com/ArmaLife/Framework/issues/646. 

**If you are fluent in one or more of these languages could you please verify their accuracy.** 

@danielstuart14 @setoy @NiiRoZz @Shield1739 @DardoTheMaster

### CODE REVIEW

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -7279,25 +7279,25 @@
...
-        </Key>
-		<Key ID="STR_Process_Partial">
+            <Czech>Nemáš zásob prostor ke zpracování vašich materiálů.</Czech>
+            <French>Vous avez pas d'espace d'inventaire pour traiter vos matériaux.</French>
+            <Spanish>Usted no tiene espacio en el inventario para procesar sus materiales.</Spanish>
+            <Italian>Non hai spazio nell'inventario per elaborare i materiali.</Italian>
+            <Polish>Nie masz miejsca na przetwarzanie swoich zapasów materiałów.</Polish>
+            <Portuguese>Você não tem espaço no inventário para processar seus materiais.</Portuguese>
+            <Russian>У вас нет места инвентаря для обработки ваших материалов.</Russian>
+            <German>Sie haben keinen Platz im Inventar Ihre Materialien zu verarbeiten.</German>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Du hast keinen Platz im Inventar, um die Materialien zu verarbeiten.

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -7279,25 +7279,25 @@
...
-            <Italian></Italian>
-            <German></German>
+            <Czech>Pouze část svých materiálů by mohla být zpracována z důvodu dosažení maximální váhu.</Czech>
+            <French>Seule une partie de votre matériel pourrait être traitée en raison d'atteindre votre poids maximum.</French>
+            <Spanish>Sólo una parte de los materiales se ha podido procesar debido a alcanzar su peso máximo.</Spanish>
+            <Italian>Solo una parte del tuo materiali possono essere trattati a causa di raggiungere il vostro peso massimo.</Italian>
+            <Polish>Tylko część swoich materiałów może być przetwarzane ze względu na osiągnięcie maksymalnej wadze.</Polish>
+            <Portuguese>Apenas uma parte de seus materiais poderia ser processado devido a atingir o seu peso máximo.</Portuguese>
+            <Russian>Только часть ваших материалов может быть обработан из-за достижения вашего максимального веса.</Russian>
+            <German>Nur ein Teil Ihrer Materialien könnten aufgrund der Erreichung Ihrer maximalen Gewicht verarbeitet werden.</German>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: German: Nur ein Teil der Materialien kann verarbeitet werden, da das maximale Gewicht sonst erreicht würde.

File: Altis_Life.Altis/stringtable.xml (hunk 1 above)
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Fixed. Thanks!

File: Altis_Life.Altis/stringtable.xml (hunk 2 above)
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Fixed. Thanks!

**Comment by setoy** [comment]
I'll have a commit ready in a few minutes that updates a few german translations and keys. You're alright that I create a PR?

**Comment by Jawshy** [comment]
@setoy, go for it. I'll handle any conflicts.

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -973,7 +973,7 @@
             <Russian></Russian>
             <German>Du kannst nicht mehr als $999,999 überweisen</German>
             <French>Vous ne pouvez pas transférer plus de $999,999</French>
-            <Italian>Non puoi trasferire più di €999.000 alla volta</Italian>
+            <Italian>Non puoi trasferire più di $999.000 alla volta</Italian>
             <Portuguese>Você pode trasferir no máximo R$999.999.</Portuguese>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Shouldn't this also be $ insteamd of R?

Comment (danielstuart14) [inline Altis_Life.Altis/stringtable.xml]: portuguese strings use R$, that means "real", the currency in brazil.
I recommend changing the symbols for each language. But it should be done in all strings.
All portuguese strings use R$.

**Pull request merged**